kubetest2-gke/deployer/network_test.go

```diff
@@ -131,3 +131,26 @@ func TestPrivateClusterArgs(t *testing.T) {
 		})
 	}
 }
+
+func TestAssertNoOverlaps(t *testing.T) {
+	testCases := []struct {
+		ranges     []string
+		shouldPass bool
+	}{
+		{[]string{"10.0.0.0/24", "11.0.0.0/24"}, true},
```

Comment:
It's better to use named fields instead of anonymous ones here.

Reply:
Done